Paraguay/PRY_transform_load_dlt.py

```diff
@@ -163,14 +170,22 @@ def boost_bronze_municipal():
             ).withColumn('func_sub', lit(None).cast("string")
             ).withColumn('adm1_name', lit(None).cast("string")
             ).withColumn('FUNCTION2', lit(None).cast("string")
+            ).withColumn('geo_0.5', 
+            when(col('adm1_name').isin('Asuncion', 'Central'), 'Central (Asuncion, Central)')
+            .when(col('adm1_name').isin('Alto Parana', 'Caaguazu', 'Canindeyu'), 'North-East (Caaguazu, Alto Parana, Canideyu)')
+            .when(col('adm1_name').isin('Alto Paraguay', 'Amambay', 'Boqueron', 'Concepcion', 'Cordillera', 'Presidente Hayes', 'San Pedro'), 'North-West (Boqueron, Alto Paraguay, Presidente Hayes, Conception, Amambay, San pedro, Cordillera)')
+            .when(col('adm1_name').isin('Guaira', 'Misiones', 'Neembucu', 'Paraguari'), 'South-East (Guaira, Misiones, Paraguari, Neembucu)')
+            .when(col('adm1_name').isin('Caazapa', 'Itapua'), 'South-West (Caazapa, Itapua)')
+            .otherwise(None) 
```

Comment:
I also recall you suggested removing the parathesis with admin1 names to keep it short. Let's do that.